keyboards/annepro2/halconf.h

```diff
@@ -321,7 +321,7 @@
  *          buffers.
  */
 #if !defined(SERIAL_BUFFERS_SIZE) || defined(__DOXYGEN__)
-#define SERIAL_BUFFERS_SIZE         16
+#define SERIAL_BUFFERS_SIZE         32
```

Comment:
64 might make more sense as it would eat the biggest message (row update)

Reply:
Good point. but Honestly before we implement DMA in the low level driver (inside chibios) there is no real need for this I guess.

Reply:
is it currently at least interrupt-driven?

Reply:
yes, it is interrupt driven. IIRC the UART has some small FIFO buffers, I think it is a 16550-like uart device. So there are like maybe 8 bytes of FIFO for Rx and Tx.